src/gamepad/controller.cpp

```diff
@@ -15,6 +20,29 @@ void Controller::updateButton(pros::controller_digital_e_t button_id) {
     }
 }
 
+void Controller::updateScreen() {
+    if (pros::millis() - this->last_print_time < 50)
+        return;
+
+    for (int i = 1; i < 3; i++) {
+        int line = (this->last_printed_line + i) % 3;
+        
+        if (pros::millis() - this->line_set_time[line] < this->screen_contents[line].second)
+            continue;
+        
+        
+        this->controller.clear_line(line);
```

Comment:
```suggestion
```
Any calls to any controller text methods, including `clear_line`, need a delay after them. Use extra spaces at the end of `set_text` to clear instead.